### PR TITLE
chore(open-ux-tools) - update dependabot config and add script to update lock file

### PR DESCRIPTION
**Pull request by devinea**
- Update dependabot config to limit node, typescript and jest versions. 
- Run weekly instead of daily as As updates are being done manually currently as 

dependabot is not currently very useful as it is missing support for pnpm which may come in 2022.
https://github.com/dependabot/dependabot-core/issues/1736#issuecomment-1095558364

- Add script to update pnpm lock file for dependabot as workaround in the meantime.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -7,19 +7,28 @@ updates:
...
+          time: '09:00'
     - package-ecosystem: 'npm'
       directory: '/'
-      open-pull-requests-limit: 20
+      open-pull-requests-limit: 5
       rebase-strategy: 'auto'
       schedule:
           interval: 'weekly'
+          # Check for npm updates on Mondays
+          day: 'monday'
```
Comment (IainSAP) [inline .github/dependabot.yml:23]: Would the weekend be more appropriate?

**Review by kranthie-sap — APPROVED** [review]
Looks good to me. Thanks.

**Review by ullasholla — CHANGES_REQUESTED** [review]
Thanks @devinea, the changes look good.

However, does it make sense to stop dependabot from creating PRs if it not support pnpm yet?

**Comment by devinea** [comment]
> Thanks @devinea, the changes look good.
> 
> However, does it make sense to stop dependabot from creating PRs if it not support pnpm yet?

@ullasholla https://github.com/SAP/open-ux-tools/pull/496/files#diff-243ea1a69df2888daa031a562e888a97d4a5a118147aab43c8c8d26c70e26499 should provide a workaround for the missing pnpm support.

**Comment by ullasholla** [comment]
> > Thanks @devinea, the changes look good.
> > However, does it make sense to stop dependabot from creating PRs if it not support pnpm yet?
> 
> @ullasholla https://github.com/SAP/open-ux-tools/pull/496/files#diff-243ea1a69df2888daa031a562e888a97d4a5a118147aab43c8c8d26c70e26499 should provide a workaround for the missing pnpm support.

Thanks. Do you mind updating the PR description with this information? It's not very obvious otherwise

**Comment by devinea** [comment]
> > > Thanks @devinea, the changes look good.
> > > However, does it make sense to stop dependabot from creating PRs if it not support pnpm yet?
> > 
> > 
> > @ullasholla https://github.com/SAP/open-ux-tools/pull/496/files#diff-243ea1a69df2888daa031a562e888a97d4a5a118147aab43c8c8d26c70e26499 should provide a workaround for the missing pnpm support.
> 
> Thanks. Do you mind updating the PR description with this information? It's not very obvious otherwise

Updated @ullasholla

**Review by ullasholla — APPROVED** [review]
Thanks @devinea

**Comment by tobiasqueck** [comment]
@devinea it looks like you added `open-pull-requests-limit: 5` twice to the config. Could you please fix it when time permits. Absolutely NOT urgent, just like to get it fixed before change it again, and miss the 2nd setting

**Comment by devinea** [comment]
> @devinea it looks like you added `open-pull-requests-limit: 5` twice to the config. Could you please fix it when time permits. Absolutely NOT urgent, just like to get it fixed before we change it again, and miss the 2nd setting

@tobiasqueck That is correct. The limit is under 2 different objects in the config. `- package-ecosystem: 'github-actions'` and `- package-ecosystem: 'npm'`

**Comment by tobiasqueck** [comment]
@devinea sorry, I missed that.
(I hate yaml!)

**Pull request merged**